client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -12,8 +12,15 @@ import SitesMigrationTrialBadge from 'calypso/sites-dashboard/components/sites-m
 import { useSelector } from 'calypso/state';
```

Comment:
The code in this file seems repetitive of `sites-grid-item.tsx`. How can we consolidate the two?

Reply:
Made some changes in b62ea9ef623ffe97aa7c1e7789cc4c7f81c2fb3d and a6f75aaa0bf2d7e2453c93ac05bc5fb992fda932 - let me know if this looks better!